lang/zh_CN.lang

```diff
@@ -631,24 +780,23 @@ value.VIGNETTE_STRENGTH.1.40=0.9
 value.VIGNETTE_STRENGTH.1.41=1.0
 
 option.DIRTY_LENS=脏镜头
-option.DIRTY_LENS.comment=为泛光添加灰尘粒子效果。 §e[*]§r此选项需要启用泛光。
+option.DIRTY_LENS.comment=为光晕添加尘埃粒子效果。§e[*]§r需要光晕。
```

Comment:
dirty lens requires bloom, changing this to "requires lens flare" isn't correct.

Reply:
changed